apps/src/aichat/views/ChatWorkspace.tsx

```diff
@@ -304,16 +300,16 @@ const ChatWorkspace: React.FunctionComponent<ChatWorkspaceProps> = ({
             multimodalAvailable={multimodalAvailable}
           />
         )}
-        <div className={moduleStyles.buttonRow}>
-          {multimodalAvailable && (
+        {multimodalAvailable && (
+          <div className={moduleStyles.buttonRow}>
```

Comment:
Moving this logic outside of the div to prevent an empty node from showing up in the DOM.